aws-lc-rs/tests/digest_test.rs

```diff
@@ -152,10 +183,54 @@ mod digest_shavs {
         assert_eq!(expected_count, 100);
     }
 
-    shavs_tests!(SHA1, SHA1_FOR_LEGACY_USE_ONLY);
-    shavs_tests!(SHA256, SHA256);
-    shavs_tests!(SHA384, SHA384);
-    shavs_tests!(SHA512, SHA512);
+    #[allow(clippy::cast_sign_loss)]
+    fn run_monte_carlo_fips_200_test(
```

Comment:
So FIPS 180-4 is the original Hash standard and FIPS 202 is for SHA3.. Should this be `run_monte_carlo_fips_202_test` or are these test vectors from the FIPS 200 publication?

Reply:
DOH. It was a typo. Fixed in next commit.